### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by gadomski**
**Related Issue(s):**
- There's changes requested on #14 and no other open PRs.

**Description:**
Includes a clean up the changelog -- v0.2.1 was never released.

The return type of a couple of public functions was changed since v0.2.0, so this requires a MAJOR release.

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [x] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

**Review by pjhartzell — APPROVED** [review]
Just one comment.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,13 +18,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 - `stac sentinel1 create-item` now required subcommand `grd` or `rtc`
 - Support for Microsoft Azure storage: similar format to SAFE, without `.SAFE` ending to the folders and slightly different file names.
 
-## [v0.1.0] - 2021-09-04
+## [0.1.0] - 2021-09-04
 
 - Initial release!
 - Support for Sentinel1 AWS RTC public dataset
+
+[Unreleased]: https://github.com/stactools-packages/sentinel1/compare/v0.3.0...HEAD
```
[the remark this answers is not in the document]
Comment (gadomski) [reply CHANGELOG.md:36]: It doesn't need to be updated in the case we change the default branch. I _think_ that's literally the only difference/benefit.

[the remark this answers is not in the document]
Comment (gadomski) [reply CHANGELOG.md:36]: Oh also it's what's in the keep a changelog template and I've never thought about it before :-): https://github.com/olivierlacan/keep-a-changelog/blob/a4d63781eb4b78574082242ad5ad843a19572332/CHANGELOG.md?plain=1#L193

[the remark this answers is not in the document]
Comment (pjhartzell) [reply CHANGELOG.md:36]: >It doesn't need to be updated in the case we change the default branch

Ah. Yes. Thanks.

**Pull request merged**